app/src/main/java/ru/netology/fmhandroid/viewmodel/PatientViewModel.kt

```diff
@@ -1,102 +1,94 @@
 import ru.netology.fmhandroid.dto.Patient
-import ru.netology.fmhandroid.dto.PatientStatusEnum
-import ru.netology.fmhandroid.model.FeedModel
-import ru.netology.fmhandroid.model.FeedModelState
-import ru.netology.fmhandroid.repository.PatientRepositoryImp
+import ru.netology.fmhandroid.dto.Patient.Status
 import ru.netology.fmhandroid.repository.patientRepository.PatientRepository
 import ru.netology.fmhandroid.repository.patientRepository.PatientRepositoryImp
 import ru.netology.fmhandroid.util.SingleLiveEvent
 
-val emptyPatient = Patient(
-        id = 0,
-        firstName = "",
-        lastName = "",
-        middleName = "",
-        birthDate = "",
-        deleted = false,
-        status = PatientStatusEnum.EXPECTED
-)
+private var PATIENT = Patient()
 
 class PatientViewModel(application: Application) : AndroidViewModel(application) {
 
     private val patientRepository: PatientRepository =
-            PatientRepositoryImp(AppDb.getInstance(context = application).patientDao())
+        PatientRepositoryImp(
+            AppDb.getInstance(context = application).patientDao(),
+            AppDb.getInstance(context = application).admissionDao(),
+            AppDb.getInstance(context = application).noteDao()
+        )
+
+    suspend fun data(): Flow<List<Patient>> =
+        patientRepository.getAllPatientsWithAdmissionStatus(Status.ACTIVE)
+
+    private val _patientCreatedEvent = SingleLiveEvent<Unit>()
+    val patientCreatedEvent: LiveData<Unit>
+        get() = _patientCreatedEvent
 
-    val data: LiveData<FeedModel> = patientRepository.data
-            .map(::FeedModel)
-            .asLiveData(Dispatchers.Default)
+    private val _loadPatientExceptionEvent = SingleLiveEvent<Unit>()
+    val loadPatientExceptionEvent: LiveData<Unit>
+        get() = _loadPatientExceptionEvent
 
-    private val _dataState = MutableLiveData<FeedModelState>()
-    val dataState: LiveData<FeedModelState>
-        get() = _dataState
-    private val edited = MutableLiveData(emptyPatient)
-    private val _patientCreated = SingleLiveEvent<Unit>()
-    val patientCreated: LiveData<Unit>
-        get() = _patientCreated
+    private val _savePatientExceptionEvent = SingleLiveEvent<Unit>()
+    val savePatientExceptionEvent: LiveData<Unit>
+        get() = _savePatientExceptionEvent
 
     init {
         loadPatientsList()
     }
 
-    fun loadPatientsList() = viewModelScope.launch {
-        try {
-            _dataState.value = FeedModelState(loading = true)
-            patientRepository.getAllPatientsWithAdmissionStatus()
-//            _dataState.value = FeedModelState()
-        } catch (e: Exception) {
-            _dataState.value = FeedModelState(errorLoading = true)
+    fun loadPatientsList() {
+        viewModelScope.launch {
+            try {
+                patientRepository.getAllPatientsWithAdmissionStatus(Status.ACTIVE)
+            } catch (e: Exception) {
+                e.printStackTrace()
+            }
```

Comment:
Вроде так не хуже получится
```suggestion
        getAllPatientsWithAdmissionStatus(Status.ACTIVE)
```